style-dictionary2/build-theme.mjs

```diff
@@ -1,5 +1,8 @@
+import hexRgb from 'hex-rgb';
+import sd from 'style-dictionary';
```

Comment:
had to change this file to a `.mjs` file and use `import` in order to load the `hexRgb` library.